frontend/src/metabase/visualizations/echarts/cartesian/chart-measurements/index.ts

```diff
@@ -160,13 +217,22 @@ export const getChartPadding = (
 
   const yAxisNameTotalWidth =
     CHART_STYLE.axisName.size / 2 + CHART_STYLE.axisNameMargin;
-
-  if (chartModel.leftAxisModel?.label) {
-    padding.left += yAxisNameTotalWidth;
-  }
-  if (chartModel.rightAxisModel?.label) {
-    padding.right += yAxisNameTotalWidth;
-  }
+  const xAxisTickOverflow = tickDimensions.xTicksWidth / 2;
```

Comment:
Based on the previous implementation, there still should be an allowed max limit so that the chart does not squash fully on very long ticks

Reply:
That makes sense. Where in the code did you find the previous max limit?

Reply:
Done